src/Amido.Stacks.API.Swagger/Filters/BasePathFilter.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace Amido.Stacks.API.Swagger.Filters
+{
+    public class BasePathFilter : IDocumentFilter
+    {
+        public BasePathFilter(string basePath)
+        {
+            BasePath = basePath;
+        }
+
+        public string BasePath { get; }
+
+        public void Apply(OpenApiDocument swaggerDoc, DocumentFilterContext context)
```

Comment:
The variable name should reflect the type name for convention.
it should either be `OpenApiDocument openApiDoc` or `OpenApiDocument apiDoc`

Reply:
I agree, but in this case VS was throwing warnings to get me to match the parameter name with the method in the interface which is `swaggerDoc` :(